scripts/ci/operator.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+# shellcheck disable=SC2119
+
+##### subscription
+CATALOG_SOURCE="${CATALOG_SOURCE:-redhat-operators}"
+SUBSCRIPTION_NAMES=("jaeger-product" "kiali-ossm" "servicemeshoperator")
+
+createSubscription() {
+  local name=$1
+
+  echo "Create Subscription resource for $name"
+  echo
+  kubectl apply -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: "${name}"
+  namespace: openshift-operators
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: "${name}"
+  source: "${CATALOG_SOURCE}"
+  sourceNamespace: openshift-marketplace
+EOF
+}
+
+create_subscription() {
+  echo "Install any operator by creating a Subscription resource."
+  echo
+  for name in "${SUBSCRIPTION_NAMES[@]}"; do createSubscription "$name"; done
+}
+
+delete_subscription() {
+  echo "Remove any operator deployed using a Subscription resource."
+  echo
+
+  echo "Delete Subscription resource"
+  for name in "${SUBSCRIPTION_NAMES[@]}"; do kubectl delete subscription "$name" -n openshift-operators; done
+}
+
+create_other_operators() {
+  echo "Install the Jaeger and Kiali operators. You need these operators to deploy a ServiceMeshControlPlane with Jaeger tracing and Kiali enabled."
+  echo
+
+  createSubscription jaeger-product
+  createSubscription kiali-ossm
+}
+
+get_clusterserviceversion() {
+  local ns=${ns:-openshift-operators}
+  echo "Get clusterserviceversion in ${ns}"
+  echo
+  for name in "${SUBSCRIPTION_NAMES[@]}"; do kubectl get subscription "${name}" -n "${ns}" -o yaml | grep currentCSV; done
+}
+
+delete_clusterserviceversion() {
+  local ns=${ns:-openshift-operators}
+  echo "Delete all clusterserviceversion in ${ns}"
+  echo
+  kubectl delete clusterserviceversion -n "${ns}" --all
+}
+
+delete_cni() {
+  local ns=${ns:-openshift-operators}
+  echo "Delete configmap"
+  kubectl delete cm istio-cni-config -n "${ns}" --ignore-not-found
+
+  echo "Delete istio cni DaemonSet(s)"
+  kubectl delete ds istio-cni-node -n "${ns}" --ignore-not-found
+}
```

Comment:
This only deletes the `istio-cni-node` daemonset, but we now also have `istio-cni-node-v2-3` and `istio-cni-node-v2-4`. You could use a label selector to delete all of them.